LanguageTools/LanguageTools.py

```diff
@@ -347,6 +347,10 @@ def updateGUIFromSettings(self):
       self.ui.weblateDownloadUrlEdit.text = settings.value("WeblateDownloadUrl", "https://hosted.weblate.org/download/3d-slicer")
       self.ui.githubRepositoryUrlEdit.text = settings.value("GitRepository", "https://github.com/Slicer/SlicerLanguageTranslations")
 
+      # Boolean values are stored as strings in the settings.
+      # False becomes 'false' and True becomes 'true'
+      textFinderIsEnabled = settings.value("EnableFindText", True)
```

Comment:
Let's make it disabled by default.
Use slicer.util.toBool to always get Boolean.

```suggestion
      textFinderIsEnabled = slicer.util.toBool(settings().value("EnableFindText", False))
```